CenterForOpenScience/osf.io
# [ENG-2745] Clean up Sloan code

⚠️  This requires the `reindex_es6` management command run to work.
⚠️⚠️⚠️⚠️⚠️  This requires FE changes!

## Purpose

Clean-up the technical debt created by the Sloan project grant by removing old test and middleware and remapping the es6 indexes.

## Changes

- removes Sloan override middleware that override waffle
- removes extensive feature flipping from serializer.
- removes old tests

## QA Notes

Please make verification statements inspired by your code and what your code touches.
- Verify eveything still works
- Verify

What are the areas of risk?

Any concerns/considerations/questions that development raised?

## Documentation



## Side Effects

None that I know of.

## Ticket

https://openscience.atlassian.net/browse/ENG-2745
